packages/nextjs/services/database/repositories/users.ts

```diff
@@ -249,3 +248,32 @@ export async function filterValidUserAddresses(addresses: string[]): Promise<str
     .where(inArray(users.userAddress, addresses));
   return rows.map(row => row.userAddress);
 }
+
+export const CHALLENGE_POINTS = 10; // Points per accepted challenge
+export const BATCH_POINTS = 20; // Points for being in a batch
+export const BUILD_POINTS = 5; // Points for _first_ build only
```

Comment:
These are dangling a bit here. Maybe we could create something like `utils/buidl-points.ts` and export these 3 from there? And we could even add some calculations there (like totalPoints).